### ISSUE TITLE
[YCQL] Failed to SELECT from TABLE with Secondary INDEX when upgrading to 2.0.5.4

### ISSUE THREAD

**Issue by nocaway** [yugabyte/yugabyte-db#3281]
After upgrading to 2.0.5.4, a column is missing when selecting from a table that has secondary index.

```
describe a;
CREATE TABLE a (
    a_hash text PRIMARY KEY,
    a_time int,
    a_height bigint
) WITH default_time_to_live = 0;
CREATE INDEX a_idx ON a (a_height, a_hash);
select * from a WHERE a_height=1;
 a_hash                                                       | a_heigh
--------------------------------------------------------+--------------
```

**Comment by nocaway** [comment]
This has been fixed in the master branch.

**Comment by nocaway** [comment]
This fix has been backported to 2.0.5 branch.
